lib/gds-components/phase-banner/PhaseBanner.tsx

```diff
@@ -0,0 +1,29 @@
+type PhaseBannerProps = {
+  linkUrl?: string;
+};
+
+export const PhaseBanner = ({ linkUrl }: PhaseBannerProps) => {
+  return (
+    <>
+      <div className="govuk-phase-banner">
+        <p className="govuk-phase-banner__content">
+          <strong className="govuk-tag govuk-phase-banner__content__tag">
+            Beta
+          </strong>
+          <span className="govuk-phase-banner__text">
+            This is a new service
+          </span>
+          {linkUrl && (
+            <span>
+              – your{" "}
+              <a className="govuk-link" href={linkUrl}>
```

Comment:
TBC. Do we have a link for this?

Reply:
Yes. I havent seen the banner in any of the new designs except [in this sketch](https://www.figma.com/file/u68018Pmi1dkVYnqUBfrkZ/DLUHC-New-Local-Plans-timetable-dashboard-concepts?type=design&node-id=3571-4238&mode=design&t=zLcsBuCTe5kkB0Lp-0) which seems pretty old

Reply:
I'll ask but for now, we can just get the banner in place

Reply:
I changed so that the text is displayed without link

Reply:
link removed completely